shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/utility/LocationUtil.kt

```diff
@@ -0,0 +1,29 @@
+package com.kcc.kmmhackathon.shared.utility
+
+import kotlin.math.*
+
+typealias Radian = Double
+typealias Degree = Double
+
+class LocationUtil {
+
+    fun getDistance(lat1: Double, lon1: Double,lat2: Double, lon2: Double, inMiles: Boolean): Double {
```

Comment:
This is annoying because both iOS and Android have built-in distance measurers using their Location and CLLocation classes. Would be great to utilise the built-in stuff if possible since we know they work perfectly, it would mean injecting into this class though

Reply:
Not sure the best way to inject stuff into KMM.  Would think we need to use the android Location, and then something like Dagger to inject it ? (V rusty on this side of the pond).

Reply:
Don't worry about it for now! We can figure it out another time. I don't think we'd _need_ to use Dagger (or Hilt) but it's usually what people use, so we can have a go.